src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -218,6 +224,104 @@ public Project GetSolutionItemsProject()
             return solutionItemsProject;
         }
 
+        public IEnumerable<Project> GetSelectedProjects()
+        {
+            var dte = this.serviceProvider.GetService<DTE>();
+            if (dte == null)
+            {
+                yield break;
+            }
+
+            foreach (object projectObj in dte.ActiveSolutionProjects as Array ?? new object[0])
+            {
+                var project = projectObj as Project;
+                if (project != null)
```

Comment:
No, because we're trying to actually change the properties which will change whether or not these properties are filtered out or not.